stdlib/3/os/__init__.pyi

```diff
@@ -396,16 +399,84 @@ def putenv(__name: Union[bytes, str], __value: Union[bytes, str]) -> None: ...
 if sys.platform != "win32":
     def unsetenv(__name: Union[bytes, str]) -> None: ...
 
-# Return IO or TextIO
+_Opener = Callable[[str, int], int]
+@overload
+def fdopen(
```

Comment:
Note for reviewer: These are all basically copied from `builtins.open()`, except for the first argument.